lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -123,7 +136,9 @@ async def _get_subscribers(self) -> [SubscriberData]:
         SUBSCRIBER_SYNC_LATENCY.observe(
             time_elapsed.total_seconds() * 1000,
         )
-        return subscribers
+
+        self._store.update_digest(digest.md5_base64_digest)
```

Comment:
To me, "get subscribers" is a read-only action. So it should return the updated digest, and then either "process subscribers" or something like "process digest" should be the one to update/handle the digest